### ISSUE TITLE
Error 3006

### ISSUE THREAD

**Issue by MaelSholjhaes** [ShikyoKira/Project-New-Reign---Nemesis-Main#528]
ERROR(3006): Invalid file name. Please enter a file name that is within the format. Please contact the mod author
Project: DefaultFemale.txt~1
Header: 1157.txt

Perhaps something to do with Vortex?

**Comment by DokuseiSan** [comment]
No, it's going to be to do with what the mod author named the file. If the file in question is that *.txt~ try removing the `~`
